src/test/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/ccd/EventTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class EventTest {
+
+    @Test
+    public void has_correct_values() {
+        assertEquals("startAppeal", Event.START_APPEAL.toString());
```

Comment:
This test will need changing if you remove some events.

Reply:
Removed all other events except submitAppeal